src/app/templates/activity/Activity.tsx

```diff
@@ -66,7 +66,7 @@ export const ActivityComponent: React.FC<Props> = ({ assetSlug }) => {
   return (
     <div className="w-full max-w-sm mx-auto">
       <div className={classNames('my-3 flex flex-col', popup && 'mx-4')}>
-        {loading && activities.length === 0 && <div className="w-full mb-4 flex justify-center">{promotion}</div>}
+        {!loading && activities.length === 0 && <div className="w-full mb-4 flex justify-center">{promotion}</div>}
```

Comment:
Sorry, but we were going to show this ad while the operations are being loaded. Did you agree on this change with Dmytro Rusnak?

Reply:
We agreed on placing ads on the top (not inside the activity list)